app/views/mail_views.py

```diff
@@ -0,0 +1,36 @@
+
+""""""
+from flask import request, jsonify
+import datetime
+from app.models.mail_model import Messages
+from flask.views import MethodView
+from app.views.validations import Validations
+from app.models.mail_model import message_list
+
+
+class CreateMail(MethodView):
```

Comment:
This is some nice stuff you have going on. 
Kudos!

Reply:
Thanks